test/quantization/test_numeric_suite_fx.py

```diff
@@ -231,3 +232,86 @@ def compare_and_validate_results(float_model, q_model, module_swap_list, data):
         compare_and_validate_results(
             backup_prepared_model, q_model, module_swap_list, linear_data
         )
+
+    @skipIfNoFBGEMM
+    def test_compare_model_outputs_conv_static_fx(self):
+        r"""Compare the output of conv layer in static quantized model and corresponding
+        output of conv layer in float model
+        """
+
+        def calibrate(model, calib_data):
```

Comment:
same here, you can use https://github.com/pytorch/pytorch/blob/master/torch/testing/_internal/common_quantization.py#L81

Reply:
Done